renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "commitBody": "Generated by renovateBot",
+  "github-actions": {
+    "fileMatch": ["^ci\/workflows\/[^/]+\.ya?ml$"]
```

Comment:
```suggestion
    "fileMatch": ["^ci\\/workflows\\/[^/]+\\.ya?ml$"]
```
?

Reply:
Honestly, idk. Do they have an example in their docs somewhere?

Reply:
In JSON's spec, updated one should be correct.

Reply:
With updated one, `cat renovate.json | jq -r '.["github-actions"].fileMatch[0]'` returns `^ci\/workflows\/[^/]+\.ya?ml$`, so, this must be correct.